docs/modeling.md

```diff
@@ -538,8 +538,35 @@ s.solve(cp_model_probing_level = 2,
 
 See [the API documentation of the solvers](api/solvers.html) for information and links on the parameters supported. See our documentation page on [solver parameters](solver_parameters.html) if you want to tune your hyperparameters automatically. 
 
+## Accessing the underlying solver object
```

Comment:
Maybe better to put this into a new file and add a link to it in the section "Solving a model" as it already mentions our own status object?

Reply:
a new file under advanced guides you mean?

Reply:
I actually like that it is just grouped together with the other use cases for using the solver interface